pypeit/edgetrace.py

```diff
@@ -1473,18 +1380,17 @@ def flagged_bits(self):
                     else np.unique(np.concatenate([self.bitmask.flagged_bits(b) 
                                                     for b in np.unique(self.spat_msk)])).tolist()
 
+    # TODO: Break the ginga commands out into a separate method?
     ## TODO It is confusing that this show routine shows images flipped from the PypeIt convention.
     ## It should be rewritten to show images with spectral direction vertical like all our other QA.
-    def show(self, traceid=None, include_error=False, thin=1, in_ginga=False, include_img=False,
-             include_sobel=False, img_buffer=100, flag=None, idlabel=False):
+    def show(self, include_error=False, thin=1, in_ginga=False, include_img=False,
```

Comment:
Personally, I would rather not because the only time I would actually use the method with ginga is when I run `pypeit_chk_edges`.  But I'll bend to majority opinion on this.  @profxj ?

Reply:
My vote is for ginga here. If you don't like ginga speakup and suggest that we adopt an alternative as I have done on multiple occasions (I hate ginga).  Otherwise I think we need to use ginga and stop doing things with matplotlib unless we are writing QA files to disk.

Reply:
I vote we punt on this.